packages/core/useAnimate/index.browser.test.ts

```diff
@@ -37,4 +37,42 @@ describe('useAnimate', () => {
     })
     wrapper.unmount()
   })
+
+  it('should support keyframes refs', async () => {
```

Comment:
```suggestion
  it.fails('should support keyframes refs', async () => {
```
If we commit it as failing we can merge this first into main and than into https://github.com/vueuse/vueuse/pull/4619 to see if this is working :)

Reply:
it is. i had the changes from the other PR locally to prove it out

but i don't mind either way. we can wait, or i can add `fails` and we do a follow-up PR. it may be worth just waiting tbh given we know the fix in the other PR does fix it